x-pack/plugins/security/public/management/roles/edit_role/edit_role_page.tsx

```diff
@@ -545,7 +550,12 @@ export const EditRolePage: FunctionComponent<Props> = ({
     try {
       await rolesAPIClient.deleteRole(role.name);
     } catch (error) {
-      notifications.toasts.addDanger(get(error, 'data.message'));
+      notifications.toasts.addDanger(
+        error?.data?.message ??
+          i18n.translate('xpack.security.management.editRole.errorSavingRoleError', {
```

Comment:
Sorry I should have clarified this point: the `i18n.translate` service needs to have unique ids for each distinct message, otherwise the build will fail

```suggestion
          i18n.translate('xpack.security.management.editRole.errorDeletingRoleError', {
```